include/openenclave/bits/sgx/sgxproperties.h

```diff
@@ -28,6 +28,13 @@ typedef struct _oe_sgx_enclave_image_info_t
 /* Max number of threads in an enclave supported */
 #define OE_SGX_MAX_TCS 32
 
+#ifdef OE_WITH_EXPERIMENTAL_EEID
+/* This is the maximum size SGX (currently) allows. When signing EEID base
+ * images we don't know the size that the final image will have, so chose
+ * this maximum. */
+#define EEID_SECS_SIZE 68719476736
```

Comment:
A few things:

* Can you add to the comment (a) why this number and (b) what this is in GB
* Can you write this in hex instead? It looks like a random number in decimal

Reply:
Sure, done!